prow/prowjobs/GoogleContainerTools/kpt-config-sync/kpt-config-sync-periodics.yaml

```diff
@@ -327,7 +327,27 @@ periodics:
       - 'GKE_RELEASE_CHANNEL=regular'
       - 'E2E_CLUSTER_PREFIX=standard-regular-gcenode'
       - 'E2E_NUM_CLUSTERS=1'
-      - 'E2E_ARGS=--gcenode -run=TestGCENode'
+      - 'E2E_GIT_PROVIDER=csr'
+      - 'E2E_ARGS=--gcenode -run=TestGCENode*'
```

Comment:
The `-run` regex is no longer needed, right?

Reply:
It is not required, but having the regex here makes the testgrid cleaner. 
With the regex, the testgrid only shows the selected tests. Otherwise, it shows other tests as skipped.

Reply:
It seems like it could be a gotcha, given that with the "if and only if" logic a developer might assume they just need to set the `GCENodeTest` option and the test will be run. But it reality, they need to both set the option and have a specific name format for the test itself.

It also seems okay to display the skipped tests in the testgrid, but I don't feel too strongly

Reply:
SG. I agree with additional regex adds an assumption on the test names.

The testgrid also providers an option to exclude filter by regex. We can use that to filter out the skipped test.

Reply:
done